tests/data_group_test.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+import scipp as sc
+
+
```

Comment:
Missing tests for dims,shape,etc

Reply:
See https://github.com/scipp/scipp/tree/datagroup-tests and https://github.com/scipp/scipp/issues/2951